.github/workflows/test-run-codebuild.yml

```diff
@@ -0,0 +1,42 @@
+name: Test run-codebuild ci script
+
+on:
+  pull_request:
+    paths:
+      - 'ci/scripts/run-codebuild/**'
+  push:
+    branches:
+      - master
+    paths:
+      - 'ci/scripts/run-codebuild/**'
+
+jobs:
+  test-script-run-codebuild:
+    runs-on: ubuntu-20.04
+    name: Test script run-codebuild
+    env:
+      working-directory: ci/scripts/run-codebuild
+
+    strategy:
+      matrix:
+        node-version:
+          - "12.22.7"
+          - "14"
+          - "16"
```

Comment:
Why three node versions? And will the steps below be run three times as a result?

Reply:
We are testing for future upgrades (given we know we are moving shortly to either 14 or 16). The tests below do run 3 times as a result, once on each node version